gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/JournalEntry.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more
+ *  * contributor license agreements. See the NOTICE file distributed with this
+ *  * work for additional information regarding copyright ownership. The ASF
+ *  * licenses this file to you under the Apache License, Version 2.0 (the
+ *  * "License"); you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations under
+ *  * the License.
+ *
+ */
+package org.apache.knox.gateway.services.token.impl;
+
+public interface JournalEntry {
+    String getTokenId();
+
+    String getIssueTime();
+
+    String getExpiration();
+
+    String getMaxLifetime();
```

Comment:
In general, I'm not a big fan of Javadoc bit I feel it'd beneficial to add some description for these methods (e.g max lifetime indicates a date representation or a duration in secs/hours,...).

Reply:
Agreed.